.github/workflows/pckdown.yaml

```
@@ -0,0 +1,116 @@
+        with:
+          use-public-rspm: true
+          
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        if: github.event.action != 'closed'
+        with:
+          extra-packages: any::pkgdown, local::.
+          needs: website
+
+      - name: Create _pkgdown.yml
+        if: github.event.action != 'closed'
+        run: |
```

actionlint (1.7.9)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)